tests/test_hypermap.py

```diff
@@ -31,3 +31,8 @@ def test_hypermap():
     assert adder((1, 2), (10, 20)) == (11, 22)
     assert (adder(numpy.ones((2, 2)), numpy.ones((2, 2)))
             == 2 * numpy.ones((2, 2))).all()
+
+
+def test_arithmetic_data():
+    assert Point(1, 2) + Point(10, 20) == Point(11, 22)
+    assert Point(1, 2) + 10 == Point(11, 12)
```

Comment:
This looks like it doesn't belong here.  Was it missing from the previous arithmetic PR?

Reply:
Kind of, yeah. It actually ends up covering `Elemwise.__call__` that was also added in this PR, though I forgot to mention it.

Reply:
Ok